src/compute/src/render/reduce.rs

```diff
@@ -720,8 +726,9 @@ where
     // We arrange the inputs ourself to force it into a leaner structure because we know we
     // won't care about values.
     let partial = collection
-        // TODO(#16549): Use explicit arrangement
-        .consolidate()
+        .map(|k| (k, ()))
+        .arrange_named::<RowKeySpine<_, _, _>>("Arranged ReduceMonotonic input")
+        .as_collection(|d: &(Row, Vec<Row>), _| d.clone())
```

Comment:
Following up on Jan's comment, one could alternatively write fragments like this as:
```suggestion
        .map(|k| (k, ()))
        .arrange_named::<RowKeySpine<(Row, Vec<Row>), _, _>>("Arranged ReduceMonotonic input")
        .as_collection(|d, _| d.clone())
```
This might make it easier to abstract the fragment into a `CollectionExt` method. No strong opinions on naming from my side; a few suggestions: `consolidate_stacked`, `consolidate_cols`, `consolidate_one`.

Reply:
We might even get away with calling it `consolidate`! As long as we don't import `differential_dataflow::operators::Consolidate` there shouldn't be any naming conflicts.

Reply:
I'll add a method to DD that allows the trace type as a generic argument.

Reply:
https://github.com/TimelyDataflow/differential-dataflow/pull/376

Reply:
It's been merged! Thanks @frankmcsherry